### DISCUSSION TITLE
Stripe Fiat-to-crypto onramp

### DISCUSSION THREAD

**Discussion by vishnuhifx** [flutter-stripe/flutter_stripe#1243] [General]
Could anyone let me know if the Stripe Fiat-to-crypto onramp is available in the Flutter Stripe SDK?

**Comment by remonh87** [comment]
I saw that this component is only available on the web and as of now it is not part of the sdk in Flutter. You can of course integrate the stripe js implementation in your flutter web app.

**Reply by vishnuhifx** [reply]
Ok. Thank you for the reply.
